### DISCUSSION TITLE
Where to put UseFileDeleteV2 and UseRegDeleteV2  ?

### DISCUSSION THREAD

**Discussion by shearercole** [sandboxie-plus/Sandboxie#2075] [Documents and notes]
Hi,

Are `UseFileDeleteV2=y` and `UseRegDeleteV2=y`  meant to be put into the global settings ini  or under each sandbox's ini?

I'd appreciate if this is also clarified in the respective release notes. And if it is a global setting, whether driver reload is needed. Thanks!

**Comment by DavidXanatos** [comment]
Well it works in boot places, 

globally it applyes to all boxes that dont have an individual setting
and in a box section it applyes only to tht box and overrules global options
no driver reload needed.

The general rule is that all settings work this way,
only very few settings need to be in the global section and then they are likely to require a driver reload.

Although for WFP, and object filtering I added made a real time reload.

The SandboxieLogon should be set glboally but can be set per box and does not need a reload.

**Reply by bastik-1001** [reply]
So basically, when you want it to apply globally, you add those to the global settings, and if you, for whatever reason, only want to apply them to specific boxes, you add them to their settings,

It's useful to know that SandboxieLogon=y can be applied per box, as well. This makes it possible to use that feature for boxes, even with conflicting AV software installed, when those products only mess with browsers.

**Comment by shearercole** [comment]
Thanks for the clarification. As a general practice, I reload driver when changing any global settings.
